manuskript/numpy.rst

```diff
@@ -223,7 +223,7 @@ eine gewisse Sorgfalt, und man ist für eventuelle Fehler selbst verantwortlich.
 Für die Anwendung ist es wichtig zu wissen, dass die Manipulation der Attribute
 ``shape`` und ``strides`` nicht die Daten im Speicher verändert. Es wird also
 nur eine neue Sicht auf die vorhandenen Daten vermittelt. Dies ist insofern von
-Bedeutung als das Kopieren von größeren Datenmengen durchaus mit einem größeren
+Bedeutung als dass Kopieren von größeren Datenmengen durchaus mit einem größeren
```

Comment:
I do not agree, the construct is "insofern ... als" which is not followed by "dass". But I guess that you proposed this change to allow me to really try out the new review functionality ;-)